utils/zerotrie/examples/first_weekday_for_region.rs

```diff
@@ -211,17 +210,10 @@ fn black_box<T>(dummy: T) -> T {
     }
 }
 
-#[no_mangle]
-fn main(_argc: isize, _argv: *const *const u8) -> isize {
-    icu_benchmark_macros::main_setup!();
-
+fn main() {
     // Un-comment to re-generate the bytes (printed to the terminal)
     // let trie_phf = DATA.iter().copied().collect::<ZeroTriePerfectHash<Vec<_>>>();
     // assert_eq!(trie_phf.as_bytes(), TRIE_PHF.as_bytes());
 
-    if black_box(TRIE_PHF).get(b"MV") == Some(weekday::FRI) {
-        0
-    } else {
-        1
-    }
+    assert_eq!(black_box(TRIE_PHF).get(b"MV"), Some(weekday::FRI));
```

Comment:
Nit: This probably pulls in the std formatter. Maybe just use `assert!`

Reply:
Yeah, utils use `assert_eq!` in a lot of places. It means the binsize benches won't be accurate, but we can address this later.